test/actions/pulp/consumer_test.rb

```diff
@@ -8,13 +8,13 @@ class ConsumerTestBase < ActiveSupport::TestCase
     include VCR::TestCase
 
     let(:uuid) { 'uuid' }
-    let(:consumer_name) { 'gregor' }
+    let(:name) { 'name' }
```

Comment:
I would leave this change in. If you define a var named 'name', it will make the test auto-pass every time.